org.eclipse.mylyn.releng/launcher/mylyn verify.launch

```diff
@@ -13,7 +13,7 @@
         <listEntry value="dash.fail=false"/>
     </listAttribute>
     <stringAttribute key="M2_RUNTIME" value="EMBEDDED"/>
-    <booleanAttribute key="M2_SKIP_TESTS" value="false"/>
+    <booleanAttribute key="M2_SKIP_TESTS" value="true"/>
```

Comment:
Perhaps, this is unrelated

Reply:
Yeah, I keep switching between no tests/multi threaded. 
The files keeps getting caught with everything else. 
Need a way of having personalized launch files somehow